commit ec27bcd3f688abff5b02769760763fe5b358ceae
Author: Tony Lindgren <tony@atomide.com>
Date:   Mon Jan 7 09:52:43 2019 -0800

    ARM: dts: omap4-droid4: Fix typo in cpcap IRQ flags
    
    [ Upstream commit ef4a55b9197a8f844ea0663138e902dcce3e2f36 ]
    
    We're now getting the following error:
    
    genirq: Setting trigger mode 1 for irq 230 failed
    (regmap_irq_set_type+0x0/0x15c)
    cpcap-usb-phy cpcap-usb-phy.0: could not get irq dp: -524
    
    Cc: Sebastian Reichel <sre@kernel.org>
    Reported-by: Pavel Machek <pavel@ucw.cz>
    Tested-by: Pavel Machek <pavel@ucw.cz>
    Signed-off-by: Tony Lindgren <tony@atomide.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm/boot/dts/motorola-cpcap-mapphone.dtsi b/arch/arm/boot/dts/motorola-cpcap-mapphone.dtsi
index ddc7a7bb33c0..f57acf8f66b9 100644
--- a/arch/arm/boot/dts/motorola-cpcap-mapphone.dtsi
+++ b/arch/arm/boot/dts/motorola-cpcap-mapphone.dtsi
@@ -105,7 +105,7 @@
 			interrupts-extended = <
 				&cpcap 15 0 &cpcap 14 0 &cpcap 28 0 &cpcap 19 0
 				&cpcap 18 0 &cpcap 17 0 &cpcap 16 0 &cpcap 49 0
-				&cpcap 48 1
+				&cpcap 48 0
 			>;
 			interrupt-names =
 				"id_ground", "id_float", "se0conn", "vbusvld",
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1062_ec27bcd3f688abff5b02769760763fe5b358ceae.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1062_ec27bcd3f688abff5b02769760763fe5b358ceae.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
